doc/md/getting-started.md

```diff
@@ -224,6 +245,17 @@ func (Car) Fields() []ent.Field {
 		field.Time("registered_at"),
 	}
 }
+```
+
+```go
+// <project>/ent/schema/group.go
+
+import (
+	"regexp"
+	
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+)
```

Comment:
mm.. can you explain please why? I need to see it on the web before merging it. 
Maybe removing the `import`s will make the code blocks smaller.

Reply:
I want to make it look more consistent, in some cases, we show imports, some of them we don't...